zingolib/src/lightclient/test_features.rs

```diff
@@ -26,21 +27,23 @@ impl LightClient {
     /// # Panics
     ///
     /// Panics if the address, amount or memo conversion fails.
-    pub async fn do_propose_test_only(
+    pub async fn propose_test_only(
         &self,
         address_amount_memo_tuples: Vec<(&str, u64, Option<&str>)>,
-    ) -> Result<String, String> {
-        let _receivers =
+    ) -> Result<TransferProposal, ProposeSendError> {
+        let receivers =
             receivers_from_send_inputs(address_amount_memo_tuples, &self.config().chain);
-        unimplemented!()
+        let request = transaction_request_from_receivers(receivers)
+            .expect("should be able to create a transaction request as receivers are valid.");
+        self.propose_send(request).await
```

Comment:
implemented! and great return types! but should it be called propose_send_test_only?